W3ChampionsIdentificationService.Tests/TestsAppConfig.cs

```diff
@@ -4,8 +4,10 @@ namespace W3ChampionsIdentificationService.Tests
 {
     public class TestsAppConfig : IAppConfig
     {
-        public string MongoConnectionString => "mongodb://157.90.1.251:3712"; // "mongodb://localhost:27017";
```

Comment:
this is a publically available test DB, its fine to keep this IP here, you can just swap the comments

Reply:
sure, but it didn't work for me, and I thought that it's not public url, I would return it back